rs/messaging/src/routing/stream_handler/tests.rs

```diff
@@ -639,6 +639,7 @@ fn with_induct_loopback_stream_setup(
     certification_version: CertificationVersion,
     test_impl: impl FnOnce(StreamHandlerImpl, ReplicatedState, MetricsFixture),
 ) {
+    assert!(certification_version <= CURRENT_CERTIFICATION_VERSION);
```

Comment:
I'm not convinced this is needed. What is the problem with an (unlikely, but conceivable) test for compatibility against a future version?

Same below.

Reply:
I guess it should be `MAX_SUPPORTED_CERTIFICATION_VERSION`. Well I figured we may add a certification version but no or incomplete support at some point to reserve a version or something and then this probably shouldn't be used with it. Though it would just fail in that case and it's not called with that either so I guess I can also just remove it.